uniconnect-epfl/uniconnect
# Solving interest conflict

# What I did
Fixed conflicts with PR #127 . Now the interests of the user are stored in the user database, while fetching them from the interest database

# How I did it
Adapted the InterestsScreen logic so it worked with the implementation in PR #127 

# How to verify it
Create a new account and check if the interests appear under your user document in the database


# Pre-merge checklist
The changes I have introduced:
